source/client-side-encryption/client-side-encryption.rst

```diff
@@ -237,7 +237,9 @@ MongoClient Changes
       // Implementation details.
       private mongocrypt_t libmongocrypt_handle; // Handle to libmongocrypt.
       private Optional<MongoClient> mongocryptd_client; // Client to mongocryptd.
-      private Optional<MongoClient> keyvault_client; // Optional external client containing the key vault collection.
+      private MongoClient keyvault_client; // Client used to run find on the key vault collection. This is either an external MongoClient or internal_client.
```

Comment:
If I had been a reviewer on the original spec, I would have asked that this be named `key_vault_client`, but since it predates this change, I'll accept it as it is